1.13.6-alpine/docker-entrypoint.sh

```diff
@@ -9,20 +9,21 @@ export NGINX_SUBFOLDER=${NGINX_SUBFOLDER:-0}
 export NGINX_SUBFOLDER_ESCAPED=$(echo ${NGINX_SUBFOLDER} | sed 's/\//\\\//g')
 export NGINX_OSB_BUCKET=${NGINX_OSB_BUCKET}
 export NGINX_OSB_RESOLVER=${NGINX_OSB_RESOLVER:-8.8.8.8}
+export DRUPAL_PUBLIC_FILES_PATH=${DRUPAL_PUBLIC_FILES_PATH:-sites/default/files}
```

Comment:
Are you sure it shouldn't be `web/sites/default/files`?

Reply:
sure